workflows/metatdenovo.nf

```diff
@@ -72,7 +72,6 @@ ch_multiqc_custom_methods_description = params.multiqc_methods_description ? fil
 // MODULE: local
 //
 include { MEGAHIT_INTERLEAVED              } from '../modules/local/megahit/interleaved.nf'
-include { UNPIGZ as UNPIGZ_FASTA_PROTEIN   } from '../modules/local/unpigz.nf'
```

Comment:
You moved this to the new subworkflow, but never used it. This makes me wonder which output files from the three orf callers are gzipped and which are not; and do we need to do something about this?

Reply:
it seems that transdecoder still needs to unpigz